### PR TITLE
Add simple function to get the holiday for this day

### PR DESCRIPTION
**Pull request by zachweix**

### CODE REVIEW

**Comment by tam5** [comment]
Hi, I think this addition is a good one and will be useful.

I have a couple general questions first:

1. Do you mind providing a few examples of how this new getter would be used? Reasons being
    1. curiosity
    2. test cases
    3. documenation

2. Sort of related to 1, but what do you think of returning an array instead of a string. This might return all the relavent information for a given day. For example, I see "Rosh Chodesh Chanuka" was treated as its own "holiday", even though it is really just two coinciding days. I'm not sure about this, because I don't know yet how this would be best used, so please consider.

Finally, before merging I would want to add a few tests, and get the coverage back up to 100, as well as update the docs. I can do this if need be, but if you feel up for it then great.

I also might want to move the definitions for each holiday out of a static method and instead to be defined more similarly to the way we have the parshios defined.

File: src/Moadim/Moadim.php [hunk 1]
```diff
@@ -10,4 +10,63 @@
...
+                            return $english;
+                        case 'Hebrew':
+                            return $hebrew;
+                        default:
+                            return false;
+                    }
+                    
+                }
+            } catch (\Exception $e) {
+                // isPurimKattan will fail in non-leap years, so we can just ignore it
```
Comment (tam5) [inline src/Moadim/Moadim.php]: can we check if it is a leap year and the exception?

maybe can rethink this a bit

Comment (zachweix) [reply src/Moadim/Moadim.php]: I will update to check the exception and will only catch it for non-leap year fails and allow all other exceptions to be thrown

**Comment by zachweix** [comment]
I currently implement a method like this so that when I show a calendar, I can show information about that day, including the holiday as well as the Parsha. When viewing the monthly calendar you can see all holidays at a glance, which only seems possible by checking every holiday for every day

**Pull request merged**